Realm/Realm/Realm.csproj

```diff
@@ -58,18 +58,35 @@
     <None Include="..\..\wrappers\build\Darwin\$(Configuration)\librealm-wrappers.dylib">
       <Pack>true</Pack>
       <PackagePath>runtimes\osx\native</PackagePath>
-      <Link>wrappers\Darwin\%(Filename)%(Extension)</Link>
+      <Link>wrappers\Darwin\librealm-wrappers.dylib</Link>
     </None>
     <!-- Linux -->
     <None Include="..\..\wrappers\build\Linux\$(Configuration)\librealm-wrappers.so">
       <Pack>true</Pack>
       <PackagePath>runtimes\linux-x64\native</PackagePath>
-      <Link>wrappers\Linux\%(Filename)%(Extension)</Link>
+      <Link>wrappers\Linux\librealm-wrappers.so</Link>
     </None>
     <!-- Android -->
     <Content Include="wrappers-props\monoandroid\wrappers.props" Pack="true" PackagePath="build\monoandroid\Realm.props" />
     <None Include="..\..\wrappers\build\Android\$(Configuration)-armeabi-v7a\librealm-wrappers.so">
       <Pack>true</Pack>
+      <PackagePath>native\android\armeabi-v7a</PackagePath>
+      <Link>wrappers\Android\armeabi-v7a\librealm-wrappers.so</Link>
+    </None>
+    <None Include="..\..\wrappers\build\Android\$(Configuration)-arm64-v8a\librealm-wrappers.so">
+      <Pack>true</Pack>
+      <PackagePath>native\android\arm64-v8a</PackagePath>
+      <Link>wrappers\Android\arm64-v8a\librealm-wrappers.so</Link>
+    </None>
+    <None Include="..\..\wrappers\build\Android\$(Configuration)-x86\librealm-wrappers.so">
+      <Pack>true</Pack>
+      <PackagePath>native\android\x86</PackagePath>
+      <Link>wrappers\Android\x86\librealm-wrappers.so</Link>
+    </None>
+    <None Include="..\..\wrappers\build\Android\$(Configuration)-x86_64\librealm-wrappers.so">
+      <Pack>true</Pack>
+      <PackagePath>native\android\x86_64</PackagePath>
+      <Link>wrappers\Android\x86_64\librealm-wrappers.so</Link>
```

Comment:
Why do we need these changes?

Reply:
We can probably revert this entire file.

Reply:
Yes, VS tries to be too clever :)